Week-7/parenthesischeck.js

```diff
@@ -0,0 +1,49 @@
+const parenthesisCheck = function (arr) {
+  //   let stk_1 = []; // for {}
+  //   let stk_2 = []; // for ()
+  let stk = []; // for []
+
+  for (let e of arr) {
+    let top = stk[stk.length - 1]; // assign the top to check the opening parenthesis
+    switch (e) {
+      case "{":
+      case "[":
+      case "(":
+        stk.push(e);
+        break;
+      case "}":
+        if (stk.length !== 0 && top === "{") {
```

Comment:
@rajeshvlega9 Logic / approach looks good to me. But this could have been done simpler, can you think about the same